tests/cc13b/input.dat

```diff
@@ -0,0 +1,50 @@
+#! Tests RHF CCSD(T)gradients
+memory 500 mb
+
+molecule h2o {
+0 1
+O
+H 1 R
+H 1 R 2 A
+
+R = 1.0
+A = 104.5
+}
+
+set {
+   reference rhf
+   basis cc-pvdz
+   r_convergence 13
+   e_convergence 13
+   d_convergence 13
+}
+
+analytic, wfn = gradient('ccsd(t)', return_wfn=True)
+
+set findif points 5
+fin_diff, wfn = gradient('ccsd(t)', dertype=0, return_wfn=True)
```

Comment:
@dgasmith  I have two test cases: water for checking if my code works with symmetry and N2 in C1 symmetry for checking the degeneracies handling. If I use 6-31 G basis the total wall time is around 6 seconds on 1 thread where I am comparing the results with hard-coded finite difference 5 point formula results to 10 digits. So now can I put these tests to the quick tests ??